test/util/splitPath-spec.js

```diff
@@ -0,0 +1,25 @@
+import { expect } from 'chai';
+import splitPath from '../../lib/util/splitPath';
+
+describe('splitPath', () => {
+  it('test number path', () => {
+    const path = 1;
+    const result = splitPath(path);
+    expect(result).to.deep.equal(['1']);
+  });
+  it('test number path', () => {
+    const path = 0;
+    const result = splitPath(path);
+    expect(result).to.deep.equal(['0']);
+  });
+  it('test array path', () => {
+    const path = ['foo', 'bar', 'x'];
+    const result = splitPath(path);
+    expect(result).to.equal(path);
+  });
+  it('test string path', () => {
+    const path = 'bar.0.y';
+    const result = splitPath(path);
+    expect(result).to.deep.equal(['bar', '0', 'y']);
+  });
```

Comment:
Awesome, thanks for the tests, though could you follow the existing style of tests, specifically:

* blank lines between tests
* complete sentences (when "it" is prepended) for the test names, e.g. `it('treats a number as a single step path')`